### PR TITLE
manager: pid of dockerd can be detected dynamically when testing iscsi

### PR DESCRIPTION
**Pull request by shuo-wu**

### CODE REVIEW

**Comment by yasker** [comment]
LGTM. Let’s wait for all the test passed, in case this one needs some extra touches.

**Comment by yasker** [comment]
Add https://github.com/rancher/longhorn/issues/10 to the commit message.

Remove `bug fixed.` in the commit title.

Otherwise LGTM.

File: manager/integration/tests/test_node.py [hunk 1]
```diff
@@ -959,6 +960,7 @@ def test_node_delete_umount_disks(client):  # NOQA
     subprocess.check_call(cmd)
 
 
+@pytest.mark.debug   # NOQA
```
Comment (yasker) [inline manager/integration/tests/test_node.py]: what's this?

File: manager/integration/tests/test_node.py [hunk 2]
```diff
@@ -1003,21 +1005,22 @@ def test_replica_cleanup(client):  # NOQA
...
         data_paths.append(replica["dataPath"])
 
+    dockerd_pid = find_dockerd_pid() or "1"
     # data path should exist now
     for data_path in data_paths:
-        assert subprocess.call(["nsenter",
-                                "--mount=/host/proc/1/ns/mnt",
-                                "--net=/host/proc/1/ns/net",
-                                "bash", "-c", "ls " + data_path]) == 0
+        assert subprocess.call(
```
Comment (yasker) [inline manager/integration/tests/test_node.py]: Can we use `exec_nsenter` instead?

Comment (shuo-wu) [reply manager/integration/tests/test_node.py]: No. Using exec_nsenter() will cause error.

File: manager/integration/tests/test_node.py (hunk 1 above)
Comment (shuo-wu) [reply manager/integration/tests/test_node.py]: sorry. I will delete this line

File: manager/integration/tests/test_node.py (hunk 2 above)
Comment (yasker) [reply manager/integration/tests/test_node.py]: why it would cause an error?

File: manager/integration/tests/test_ha.py [hunk 3]
```diff
@@ -250,7 +250,7 @@ def test_ha_prohibit_deleting_last_replica(client, volume_name):  # NOQA
 
     with pytest.raises(Exception) as e:
         volume.replicaRemove(name=replica0["name"])
-    assert "no other healthy replica available" in str(e.value)
+    assert "no other healthy replica available" in str(e.value).lower()
```
Comment (yasker) [inline manager/integration/tests/test_ha.py]: this line has been fixed in the manager, so shouldn't need this one.

**Comment by yasker** [comment]
LGTM.

**Pull request merged**